ChangeLog

```diff
@@ -845,7 +845,7 @@ Version 3.0.0 Release Candidate 1 - 16 June 2009
 Version 2.2.0 - 11 August 2008
 ------------------------------
  - Do not install domjudge straight in $HOME by default.
- - [security] Add sanity check in jury interface whether basic authentication
+ - [security] Add coherence check check in jury interface whether basic authentication
```

Comment:
Double check

Reply:
Indeed, let's not retroactively change ChangeLog entries unless they were initially incorrect.